lib/mix/tasks/phoenix.gen.html.ex

```diff
@@ -1,7 +1,7 @@
 defmodule Mix.Tasks.Phoenix.Gen.Html do
   use Mix.Task
 
-  @shortdoc "Generates HTML files for a resource"
+  @shortdoc "Generates controller, model, views for an HTML-based resource"
```

Comment:
I did it on purpose because I tried to not have a long sentence, as most of
the @shordocs of other tasks are quite short.

But you are right.

On Fri, Apr 24, 2015 at 4:14 PM Sonny Scroggin <notifications@github.com>
wrote:

> In lib/mix/tasks/phoenix.gen.html.ex
> <https://github.com/phoenixframework/phoenix/pull/803#discussion_r29086426>
> :
>
> > @@ -1,7 +1,7 @@
> >  defmodule Mix.Tasks.Phoenix.Gen.Html do
> >    use Mix.Task
> >
> > -  @shortdoc "Generates HTML files for a resource"
> > +  @shortdoc "Generates controller, model, views for an HTML-based resource"
>
> "controller, model, and views..."
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/phoenixframework/phoenix/pull/803/files#r29086426>.
>